tiny/src/main.rs

```diff
@@ -123,10 +123,12 @@ fn run(
         let mut clients: Vec<Client> = Vec::with_capacity(servers.len());
 
         for server in servers.iter().cloned() {
-            tui.new_server_tab(&server.addr);
+            let server_name = server.alias.clone().unwrap_or(server.addr.clone());
```

Comment:
You shouldn't need cloning here as `new_server_tab` expects a ref.

Reply:
The clone is so that we can pass the values to `ServerInfo`  on line 130 and 131.

I can try it this way, if you want: 
```
            let server_name: String = match server.alias {
                Some(alias) => alias,
                None => server.addr.clone()
            };
            // let server_name = server.alias.unwrap_or(server.addr);
            tui.new_server_tab(&server_name);
```

Reply:
Hmm, that's fair. Yet another possibility is

```
let server_name = server.alias.clone().unwrap_or_else(|| server.addr.clone());
```

This only clones `server.addr` if `server.alias` is `None`. However your code with `match` is best I think as it doesn't clone `alias` when it's not `None`. I suggest we go with the `match`.